mobx/README.md

```diff
@@ -13,56 +12,52 @@
 
 ## Building blocks
 
-### Observables 
+### Observables
 
-- [x] Create `ObservableValue<T>` via `observable<T>()`
+- [x] Create `Observable<T>` via `Observable<T>()`
 - [x] Create `ObservableList<T>`
-    - [ ] `observe` and `intercept`
-    - [ ] `onBecomeObserved` and `onBecomeUnobserved`
+  - [ ] `observe` and `intercept`
+  - [ ] `onBecomeObserved` and `onBecomeUnobserved`
 - [ ] Create `ObservableMap<K, T>`
-- [x] Atoms with `createAtom()`
+- [x] Atoms with `Atom()`
 
-### Computed Observables 
+### Computed Observables
 
-- [x] Create `ComputedValue<T>` via `computed<T>()`
+- [x] Create `Computed<T>` via `Computed<T>()`
 - [x] 2-phase change propagation
 
-### Reactions 
+### Reactions
 
 - [x] Create `Reaction` with `autorun()`
-    - [x] with `delay`
+  - [x] with `delay`
 - [x] Create `Reaction` with `reaction()`
-    - [x] with `delay`
-    - [x] with `fireImmediately`
+  - [x] with `delay`
+  - [x] with `fireImmediately`
 - [x] Create `Reaction` with `when()`
-    - [x] `when` with timeout
-    - [x] `when()` returning `Future<T>`
- 
+  - [x] `when` with timeout
+  - [x] `when()` returning `Future<T>`
 
-### Actions 
+### Actions
 
-- [x] Create `Action` with `action()`
+- [x] Create `Action` with `Action()`
 - [x] Execute within `runInAction`
 - [x] Create untracked-action with `untracked<T>()`
 - [x] Create transaction with `transaction<T>()`
 
-
 ## Cross cutting features
 
-- [x] Observability API for `ObservableValue` and `ComputedValue` 
-    - [x] `observe`
-    - [x] `intercept`
-    - [x] `onBecomeObserved`
-    - [x] `onBecomeUnobserved`
+- [x] Observability API for `ObservableValue` and `ComputedValue`
```

Comment:
Have to rename here as well `ObservableValue` => `Observable`

Reply:
Renamed